src/test/scala/edu/ie3/simona/test/common/input/PvInputTestData.scala

```diff
@@ -55,6 +58,34 @@ trait PvInputTestData
     0.95,
   )
 
+  private val operationTimeBuilder = OperationTime.builder()
+
+  private val interval = new ClosedInterval[ZonedDateTime](
+    TimeUtil.withDefaults.toZonedDateTime("2020-01-01T11:00:00Z"),
+    TimeUtil.withDefaults.toZonedDateTime("2020-01-01T14:00:00Z"),
+  )
+  private val operationTime: OperationTime =
+    operationTimeBuilder.withOperationTime(interval).build()
+
+  protected val pvInputWithQCharacteristicLimitedOperationTime = new PvInput(
```

Comment:
Have you thought about using `pvInput.copy().uuid(...).id(...).operationTime(...).qCharacteristics(...).build()`? Same for load test data

Reply:
Fair point